collector/info_schema_innodb_cmp.go

```diff
@@ -0,0 +1,98 @@
+// Scrape `information_schema.INNODB_CMP`.
+
+package collector
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	//	"github.com/prometheus/common/log"
+)
+
+const innodbCmpQuery = `SELECT * FROM information_schema.INNODB_CMP`
```

Comment:
We prefer to use explicit column names in the exporter to avoid having to assume the column list.

Reply:
Also, for consistency, we can use the lowercase table name here.

Reply:
The query's rewritten